CHANGELOG.md

```diff
@@ -1,8 +1,130 @@
+
+- **roads** layer: Pedestrian paths and piers were missing `bicycle: designated` and related tags. Now that they are exported, they are correctly also decorated as `is_bicycle_related: True`. ([#832](https://github.com/tilezen/vector-datasource/issues/832))
+
+- **roads** layer: Add additional properties `sidewalk_left` and `sidewalk_right` to all road layer features. ([#605](https://github.com/tilezen/vector-datasource/issues/605) and [#986](https://github.com/tilezen/vector-datasource/issues/986))
+
+- **roads** layer: Remove `crossing`, `sidewalk`, `sidewalk_left` and `sidewalk_right` properties for road merge. ([#993](https://github.com/tilezen/vector-datasource/issues/993))
+
+- **roads** layer: Drop meaningless `id` property on merged features. ([#952](https://github.com/tilezen/vector-datasource/issues/952))
+
+- **transit** layer: Add OpenStreetMap `source` attribution per feature. ([#935](https://github.com/tilezen/vector-datasource/issues/935))
+
+  #### DOCUMENTATION CHANGES (v1.0.0-pre2)
+
+- **documentation**: Update **earth** layer documentation to reflect additional `line` and `point` geometry types. ([#808](https://github.com/tilezen/vector-datasource/issues/808))
+
+- **documentation**: Update **boundaries** and **landuse** layer documentation to reflect barriers moving into the landuse layer. ([#932](https://github.com/tilezen/vector-datasource/issues/932))
+
+- **documentation**: Update **places** layer documentation to reflect new `kind` and `kind_detail` values, locality, suburb, quarter, state, province, region, capital, and other changes. ([#934](https://github.com/tilezen/vector-datasource/issues/934))
+
+- **documentation**: Update **buildings** layer documentation to reflect new `kind` and `kind_detail` values and `building_part` changes. ([#933](https://github.com/tilezen/vector-datasource/issues/933)) and related ([#842](https://github.com/tilezen/vector-datasource/issues/842))
+
+- **documentation**: Update **roads** layer documentation to reflect new `kind` and `kind_detail` values, and remove erroneous reference to `exit`. ([#936](https://github.com/tilezen/vector-datasource/issues/936))
+
+- **contributing**: Adding a step to create a `test_config.yaml` file. ([#1001](https://github.com/tilezen/vector-datasource/pull/1001))
+
+
+  #### INTERNAL CHANGES (v1.0.0-pre2)
+
+- **performance**: Move label centroid calculation to database to reduce network pressure on the database (post processing transform needed to much geometry) in **landuse**, **water**, **earth**, and **buildings** layers. ([#965](https://github.com/tilezen/vector-datasource/issues/965))
+
+- **performance**: Add pois indexes for OpenStreetMap polygon and point tables to improve query time. ([#983](https://github.com/tilezen/vector-datasource/issues/983))
+
+- **database**: Convert Postgres PostGIS database to `h-store` all. ([#876](https://github.com/tilezen/vector-datasource/issues/876))
```

Comment:
The wording should probably be "database to utilize osm2pgsql's `--hstore-all` option"